### PR TITLE
Build all Linux versions on a single branch

### PR DESCRIPTION
**Pull request by ocefpaf**
Git rid of the Linux branch now that we are using a matrix build in CircleCI and re-test how long it takes to build on Travis-CI (so we can request more time.)

### CODE REVIEW

**Comment by Korijn** [comment]
I think you are missing the `brew cask install xquartz` line in `.travis.yml`.

**Comment by ocefpaf** [comment]
> I think you are missing the `brew cask install xquartz` line in `.travis.yml`.

Yep. Thanks

**Comment by Korijn** [comment]
The failures seem to be different than they were in #3 ...

**Comment by ocefpaf** [comment]
> The failures seem to be different than they were in #3 ...

Yep. I can't look at this today but I'll get back to it soon. (Things are always harder when we debug using Travis-CI.)

**Comment by Korijn** [comment]
@ocefpaf can you skip OSX again and refresh this PR? I'd like to get these changes merged so at least we have linux and windows working on the master branch of this feedstock.

**Comment by ocefpaf** [comment]
I'm away from office with limited connection. I'll check this as soon as I
get back.

**Comment by Korijn** [comment]
I've gone ahead and updated this PR to just be about running all Linux builds on the same branch, and I have therefore re-skipped OS X.

**Comment by Korijn** [comment]
The build failure on Windows here is odd. Nothing was changed except for the new `toolchain` requirement.... I'm going to try and remove it.

**Comment by Korijn** [comment]
Nope, that didn't work. What the heck is going on here?!

**Comment by Korijn** [comment]
I just ran this on my Windows machine, that does not have cygwin, and the build passed...

**Comment by Korijn** [comment]
```
(C:\bld\wxpython_1508605431706\_b_env) C:\bld\wxpython_1508605431706\work\wxPython-4.0.0a3>where msgfmt 
C:\Program Files\Git\usr\bin\msgfmt.exe
```

So it looks like [Git was recently updated on AppVeyor](https://www.appveyor.com/updates/), and they added msgfmt to their `bin` folder. Jeez!

**Comment by Korijn** [comment]
nice

File: recipe/build.py.patch [hunk 1]
```diff
@@ -0,0 +1,13 @@
+diff --git a/build.py b/build.py
+index 91238f7..8adc550 100644
+--- a/build.py
++++ b/build.py
+@@ -1272,7 +1272,7 @@ def cmd_build_wx(options, args):
+ 
+     # Build the wx message catalogs, but first check that there is a msgfmt
+     # command available
+-    if findCmd('msgfmt'):
++    if findCmd('msgfmt') and not isWindows:
```
Comment (ocefpaf) [inline recipe/build.py.patch]: Ouch! Sorry you had to do that to be able to build it in AppVeyor.

**Comment by ocefpaf** [comment]
Thanks @Korijn!

File: recipe/build.py.patch (hunk 1 above)
Comment (Korijn) [reply recipe/build.py.patch]: No worries, I actually enjoy figuring this kind of stuff out (eventually). :)

**Pull request merged**